tensorrt_llm/_torch/pinned_weight_staging.py

```
@@ -0,0 +1,160 @@
+                freed = sum(b.numel() * b.element_size()
+                            for b in _bufs.values())
+                _bufs.clear()
+                # Best-effort: push the freed pinned blocks out of torch's
+                # caching host allocator so the pages return to the OS.
+                hec = (getattr(torch.cuda, "host_empty_cache", None)
+                       or getattr(torch._C, "_host_emptyCache", None))
+                if hec is not None:
+                    try:
+                        hec()
+                    except Exception:
+                        pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)